views/tx.tmpl

```diff
@@ -41,6 +41,9 @@
               <div class="d-inline-block mix-box mx-2 fs14">Mix: {{.MixCount}} outputs of {{template "decimalParts" (amountAsDecimalParts .MixDenom false)}} DCR
               </div>
               {{- end}}
+              {{if gt (len $.Swaps) 0}}
+              <div class="d-inline-block swap-box mx-2 fs14">Swap {{$.AtomicSwapTxType}}</div>
+              {{end}}
```

Comment:
How about separate labels: **Swap**, **Redemption**, and **Refund**. More than one could possibly be displayed, though it wouldn't be common. The shorter words = smaller labels wouldn't wrap there either, which would look better. Here it is at a common mobile width. 

![image](https://user-images.githubusercontent.com/6109680/80831856-271e7080-8bb1-11ea-8e2b-795fd00e990f.png)